android/library/src/main/java/com/liferay/mobile/screens/webcontent/display/view/WebContentDisplayViewModel.java

```diff
@@ -28,4 +28,11 @@
 	 * @param html {@link WebContent} content
 	 */
 	void showFinishOperation(WebContent html);
+
+	/**
+	 * Customize the {@link WebContent} html with custom CSS.
+	 *
+	 * @return css style
+	 */
+	String getCustomCssStyle();
```

Comment:
I think we have two options here:
* We can create a new property in the screenlet that searches in the project a css file to inject in the WebContent
* We can remove the method from the view model and make it open to allow developers override it in the children views
What do you think?